.eslintrc

```diff
@@ -8,7 +8,8 @@
   "parser": "babel-eslint",
   "rules": {
     "valid-jsdoc": "error",
-
+    "react/no-did-update-set-state": "warn",
+    "import/no-cycle": "warn",
```

Comment:
I'm not sure if this is normal, but the eslint on master was failing because some dependencies were defined like a cycle. 

We should take a look of this.